packages/regl-worldview/src/utils/getHitmapDefaults.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import type { AssignNextIdsFn, MouseEventObject } from "../types";
+import { intToRGB } from "./commandUtils";
+
+export const nonInstancedGetHitmap = <T: Object>(
+  props: Array<T> | T,
+  assignNextIds: AssignNextIdsFn,
+  alreadySeenObjects: Array<MouseEventObject>
```

Comment:
📝 I think it would make more sense to call this `excludedObjects` so it's clear why we care about having "seen" them.